build.gradle

```diff
@@ -249,37 +243,55 @@ dependencies {
         implementation 'org.apache.commons:commons-collections4:4.4'
         implementation 'org.openapitools:jackson-databind-nullable:0.2.2'
     } else {
-        implementation ('bio.terra:terra-common-lib:0.0.89-SNAPSHOT') {
-            exclude group: 'org.broadinstitute.dsde.workbench', module: 'sam-client_2.12'
-        }
+        implementation 'bio.terra:terra-common-lib:0.1.10-SNAPSHOT'
```

Comment:
Heads up that Doug has some work in flight here: TCL is now at 0.1.11 which, among other things, will allow us to simplify our direct dependencies on OTEL (we'll get them for free from TCL).

https://github.com/DataBiosphere/terra-common-lib/pull/131
https://github.com/broadinstitute/rawls/pull/2727

But I would hold off on bumping within this PR, it will be better IMO to get this out the door first.

Reply:
That will be great! I put in a ticket for this work - https://broadworkbench.atlassian.net/browse/DC-840